core-services/service-request/src/main/java/org/egov/servicerequest/web/models/ServiceDefinition.java

```diff
@@ -50,7 +50,7 @@ public class ServiceDefinition {
     @Valid
     private AuditDetails auditDetails = null;
 
-    @JsonProperty("additionalFields")
+    @JsonProperty("additionalDetails")
```

Comment:
https://github.com/egovernments/configs/blob/UNIFIED-DEV/health/egov-persister/service-request-persister.yml
check this @KolaganiNavya